src/js/webrequest.js

```diff
@@ -536,7 +536,7 @@ function checkAction(tabId, url, quiet, frameId){
   if (action && ! quiet) {
     badger.logTrackerOnTab(tabId, requestHost, action);
     if (_.contains(constants.BLOCKED_ACTIONS, action)) {
-      badger.updateCount(tabId);
+      setTimeout(badger.updateCount(tabId), 0);
```

Comment:
Thanks! I kept looking at this and had a sense something was wrong. I should've just brushed up on the documentation. :)